bin/merge_kraken_res.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import pandas as pd
+import numpy as np
+
+
+def _get_args():
+    '''This function parses and return arguments passed in'''
+    parser = argparse.ArgumentParser(
+        prog='merge_kraken_res',
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description='Merging csv count files in one table')
+    parser.add_argument(
+        '-o',
+        dest="output",
+        default=None,
+        help="Output file. Default = sources.csv")
```

Comment:
What does 'sources' mean here? Or is this a leftover from CoproID? Maybe should be changed to same on line 58?

Reply:
From [kraken-nf](https://github.com/maxibor/kraken-nf) actually ;)
Changed to use the name given by the CLI. (default="kraken_count_table.csv")